cluster-scope/base/apiextensions.k8s.io/customresourcedefinitions/reports.batch.curator.openshift.io/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+# This kustomization.yaml is not intended to be run by itself,
+# since it depends on service name and namespace that are out of this kustomize package.
+# It should be run by config/default
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- batch.curator.openshift.io_reports.yaml
+# +kubebuilder:scaffold:crdkustomizeresource
```

Comment:
what's this for? does it need to be in this file?

Reply:
When I was deploying it on the Smaug cluster. They mentioned crd should go through Github. 
Link - https://github.com/operate-first/apps/pull/1166/files
So, I assumed the same will be the scenario for moc-staging env.

Reply:
Sorry, I wasn't clear. I'm asking what the line that begins with `#+kubebuilder:scafold` does?

Reply:
Even I don’t know much but what I found is - ‘ when a controller is created, the main file has to include some code fragments to load it, and where to place those code fragments is marked by kubebuilder:scafold‘

This line is commented so will not impact the code. As this was generated by kubebuilder. There might be some commented lines.